sources/Tools/Visualiser.py

```diff
@@ -196,17 +200,25 @@ def _exit(exception=None, message="Error", mod=-1):
         logging.error(f"{message}")
         sys.exit(mod)
 
+    @property
+    def _get_houses(self):
+        return list(set(self.raw_data.loc[:, "Hogwarts House"]))

```

Comment:
check numpy.unique()

Reply:
i've change it without another function. good call